Docs: Explain `local_assigns` is undefined when using Strict Locals and workaround for reserved keywords (#52162)
rails/rails: guides/source/action_view_overview.md

CAUTION: Only keyword arguments are supported. Defining positional or block
arguments will raise an Action View Error at render-time.

> When using Strict Locals in a partial, `local_assigns` will become undefined. To access a local
> variable that is named the same as a reserved Ruby keyword, like `class` or `if`, those values
> can be accessed through `binding.local_variable_get`:

Reviewer: ```suggestion
When using Strict Locals in a partial, `local_assigns` will be undefined. If a
variable is named the same as a reserved Ruby keyword, like `class` or `if`, then those values
can be accessed through `binding.local_variable_get`:
```

> To access a local variable ... those values can be accessed ...

sounded a bit odd to me, does this suggestion make sense?
